### PR TITLE
fastjson、gson、jackson adapter 3.2

### PR DESCRIPTION
**Pull request by xixingya**
## What is the purpose of the change

adapte json
## Brief changelog

XXXXX

## Verifying this change

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/dubbo/issues) field for the change (usually before
  you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address
  just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit
  in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency
  exist. If the new feature or significant change is committed, please remember to add sample
  in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure
  unit-test and integration-test pass.
- [ ] If this contribution is large, please follow
  the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-serialization-extensions/dubbo-serialization-fastjson/pom.xml [hunk 1]
```diff
@@ -31,16 +31,33 @@ limitations under the License.
     <description>The fastjson serialization module of dubbo project</description>
     <properties>
         <skip_maven_deploy>false</skip_maven_deploy>
+        <dubbo.version>3.2.7</dubbo.version>
     </properties>
     <dependencies>
+        <dependency>
+            <groupId>org.apache.dubbo.extensions</groupId>
+            <artifactId>dubbo-serialization-common</artifactId>
```
Comment (songxiaosheng) [inline dubbo-serialization-extensions/dubbo-serialization-fastjson/pom.xml:39]: why use snapshot version

**Review by songxiaosheng — CHANGES_REQUESTED** [review]
please upgrade dubbo-serialization-fastjson version to 3.x release version

File: dubbo-serialization-extensions/dubbo-serialization-fastjson/pom.xml (hunk 1 above)
Comment (xixingya) [reply dubbo-serialization-extensions/dubbo-serialization-fastjson/pom.xml:39]: this is new module in this rep

**Review by songxiaosheng — APPROVED** [review]
LGTM

**Pull request merged**